sonic-utilities-data/templates/sonic-cli-gen/show.py.j2

```diff
@@ -0,0 +1,254 @@
+{% from "common.j2" import cli_name -%}
+"""
+Auto-generated show CLI plugin.
+{% if source_template is defined %}
+Source template: {{ source_template }}
+{% endif %}
+{% if source_yang_module is defined %}
+Source YANG module: {{ source_yang_module }}
+{% endif %}
+"""
+
+import click
+import tabulate
+import natsort
+import utilities_common.cli as clicommon
+
+
```

Comment:
Do we have unit tests that cover this generation and as well testing of the generated code ?

Reply:
Could you please take a look at these tests: https://github.com/Azure/sonic-utilities/blob/df1d329d7b8ddddcac5a3965c5bb990ef4cf1cf4/tests/cli_autogen_test.py